src/app/constants/Urls.ts

```diff
@@ -0,0 +1 @@
+export const BLOG_URL = 'https://devfolio.co/blog/supabald-jesse/';
```

Comment:
Can rather keep this in constants?

https://github.com/devfolioco/supabald-jesse/blob/d094f54ade4f949fc934ec51fb21ac8071772c64/src/app/utils/shared.ts#L9